drivers/python/tox.ini

```diff
@@ -0,0 +1,26 @@
+[tox]
+skipsdist = True
+envlist = py27, py34
+
+[base]
+deps =
+    PyOpenSSL
```

Comment:
`PyOpenSSL` is not a requirement, rather`twisted.internet.ssl` has a requirement on the `OpenSSL` module. So from our standpoint `twisted` is a requirement for whatever `/usr/bin/env python` comes back with. Additionally it is required to test the `twisted` flavor of the driver, but that does not work on Python3.0 through about Python3.2 (except on some versions of 3.2... shrug).

So `PyOpenSSL` should be removed, and `twisted` added (at least for non-3.0-3.2).